scripts/fix-comments.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import subprocess
+import re
+
+if len(sys.argv) != 2 or not sys.argv[1].endswith('.lean'):
+    print("usage: fix-lints.py X.lean")
+    sys.exit(1)
+
+leanfile = sys.argv[1]
+
+is_clean = subprocess.run(
+    ['git', 'status', '--untracked-files=no', '--porcelain'],
+    capture_output=True).stdout.decode().rstrip()
+
+if is_clean != "":
+    print("Certain files tracked by git have uncommitted changes.\n")
+    os.system("git status")
+    print("\n")
+    s = input("Type y to continue. ")
+    if s != 'y':
+        sys.exit(1)
+
+root_dir = subprocess.run(
+    ['git', 'rev-parse', '--show-toplevel'],
+    capture_output=True).stdout.decode().rstrip()
+
+align_files = subprocess.run(
+    ['git', 'grep', '-l', '^#align'],
+    cwd=root_dir,
+    capture_output=True)
+
+name_map = dict()
+for f in align_files.stdout.decode().splitlines():
+    contents = open(os.path.join(root_dir, f)).read()
```

Comment:
This leaks a file handle,
```suggestion
    with open(os.path.join(root_dir, f)) as fh:
        contents = fh.read()
```